src/modules/resample/link_resample.yml

```diff
@@ -0,0 +1,17 @@
+schema_version: 7.0
+type: link
+identifier: resample
+title: Resample
+version: 1
+copyright: Meltytech, LLC
+license: LGPLv2.1
+language: en
+url: http://www.mega-nerd.com/SRC/
+tags:
+  - Audio
+  - Hidden
+description: >
+  Adjust an audio stream's sampling rate.
+  
+  This link can be added to a chain to normalize audio from the producer to
+  provide the rate requested by the consumer.
```

Comment:
This part should be worded similar to `link_swresample.yml`.

Reply:
Done